### PR TITLE
Document how to publish to NPM

### PR DESCRIPTION
**Pull request by davidharting**
Cuz you know I'm gonna forget.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,19 @@ Follow the links to the NPM packages for instructions on installation and usage.
...
+
+We follow [semantic versioning](https://docs.npmjs.com/getting-started/semantic-versioning). If you're not sure whether your change is a patch, minor, or major change, consult another developer. 
+
+**Publish the change using Lerna** 
+```bash
+git checkout master
+git pull
+lerna clean
+lerna publish
+```
```
Comment (emmoistner) [inline README.md]: @davidharting Probably add block around getting added as a collaborator in npm and something about semver versioning.

Comment (davidharting) [inline README.md]: I linked to the NPM semver documentation..... Not sure what else to say about it.

Comment (austenLacy) [inline README.md]: @davidharting info about getting added as a collaborator would be useful tho

**Comment by davidharting** [comment]
Addressed reviews. Lemme know if it still needs more work!

**Pull request merged**